landscape/message_schemas/server_bound.py

```diff
@@ -830,7 +840,6 @@
                     ),
                 ),
             },
```

Comment:
To save us in future of something like this happening again, it might be useful to think whether we can anticipate needing to add new keys to any of these `KeyDict`s, and if so, set them to `strict=False`. If there's no reason to do so for any of these, just let me know.

Reply:
I think it's safe to since the `snap-services` message isn't mature yet.

The inner one `services.running` does have `strict=False`, let me add that to `services`